src/cr/cube/cubepart.py

```diff
@@ -923,6 +923,21 @@ def smoothed_dimension_dict(self):
         # from the matrix later on.
         return self._columns_dimension._dimension_dict
 
+    @lazyproperty
+    def sums(self):
+        """2D optional np.float64 ndarray of sum value for each table cell.
+
+        Cell value is `np.nan` for each cell corresponding to an inserted subtotal
```

Comment:
Are we sure that a subtotal of `sums` should be `np.nan`? I'm inclined to think not, and that they _do_ add, unlike `means`.

Reply:
u right